.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Run Python Tests
```

Comment:
I have to ask... is this going to rack up a bill with micro$oft?

Reply:
Not with our usage :)